client-cli/src/command/wallet_command.rs

```diff
@@ -25,6 +25,33 @@ impl WalletCommand {
         }
     }
 
+    fn get_mnemonics<T: WalletClient>(wallet_client: &T) -> String {
+        let mut mnemonics: String;
+        loop {
+            println!("== hd wallet setup==");
+            println!("1. create new mnemonics");
+            println!("2. restore from mnemonics");
```

Comment:
could these be done as the rest of `cli` -- the first one could be something like `client-cli wallet new --type hd --name ...`... the second one ``client-cli wallet restore ...`

Reply:
ok, i will separate as runtime option

Reply:
working..